snapshot/src/test/java/io/camunda/zeebe/snapshots/impl/FileBasedTransientSnapshotTest.java

```diff
@@ -234,33 +234,10 @@ public void shouldNotRemoveTransientSnapshotWithGreaterIdOnPersist() {
         .hasParent(pendingDir);
   }
 
-  @Test
-  public void shouldRemoveTransientDirectoryOnTakeFailure() {
```

Comment:
:question: Do we have an replacement for this test?

Reply:
Yes. There was an existing test `shouldRemoveTransientDirectoryOnTakeException()`.